app/assets/javascripts/admin.js

```diff
@@ -5,24 +5,29 @@
 //= require jquery_ujs
 //= require popper
 //= require bootstrap
+//= require vanilla_dom_ready
```

Comment:
Could we name this simply `dom_ready` since we'll end up in a place where all of our JS is _vanilla_?